requirements.txt

```diff
@@ -2,4 +2,7 @@ numpy>=1.8.0
 six>=1.9.0
 pytest>=2.9.0
 coverage>=4.0
-coveralls>=1.1
\ No newline at end of file
+coveralls>=1.1
+certifi>=2017.7.27.1
```

Comment:
I think we can remove these additions to the requirements.txt as they are optional/dynamic depending on the ssl version. We can alternatively have them in a new file `optional-requirements.txt`.

Reply:
Removed from requirements.txt